### DISCUSSION TITLE
cli

### DISCUSSION THREAD

**Discussion by hungl6844** [gekigek99/minecraft-vanilla-server-hibernation#82] [Q&A] [answered]
Is minecraft-server-hibernation dependent on a gui, or can I run it from a terminal? (My OpenSUSE server is without a gui, and I'd prefer not to have to use one...) .app files normally have guis, so I'm not asking randomly...

**Comment by gekigek99** [comment, marked as answer]
the executables you find in the releases contain also a .app file which can be used by macOS users (I don't think anyone has ever run this script on mac but who knows... :) )

This program supports **only terminal** interaction and no gui at all ahah

**Reply by hungl6844** [reply]
Oh, I'm sorry, I must have confused you (although it was due to my own stupidity). So the .bin file is for Linux, the .app file is for MacOS, and the .exe is for Windows! Thanks!

**Reply by hungl6844** [reply]
"only terminal interaction and no gui" good, that answers my question!

**Reply by gekigek99** [reply]
I would advice you to run the last version (v2.3.3) in the dev branch since the last release has a bug for modded servers

do you know how to compile it yourself? (if you don't, I can send you the binaries)

**Reply by hungl6844** [reply]
I don't have go installed, but I can figure it out! Thanks for the tip!

**Reply by gekigek99** [reply]
ok perfect!
anyway if you need help setting it up you can open an other discussion!

if you like the script please give the repository a star and share it with others!
